spring-boot-autoconfigure/pom.xml

```diff
@@ -497,5 +497,11 @@
 			<artifactId>slf4j-jdk14</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.apache.cxf</groupId>
+			<artifactId>cxf-bundle</artifactId>
+			<version>3.0.0-milestone2</version>
```

Comment:
No, not yet. None of the other comments have been addressed. In particular, I'd prefer not to need to extend `CXFAdapter` or, at the very least, to understand why it's necessary.